cmd/generate/config/rules/settlemint.go

```diff
@@ -0,0 +1,58 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func SettlemintPersonalAccessToken() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Found a Settlemint Personal Access Token.",
+		RuleID:      "settlemint-personal-access-token",
+		Regex:       utils.GenerateUniqueTokenRegex(`(sm_pat_)[a-zA-Z0-9]+`, false),
+		Keywords: []string{
```

Comment:
I'd recommend adding entropy to each rule. 3 is usually a good baseline but YMMV.
```suggestion
                Entropy: 3,
		Keywords: []string{
```

Reply:
Ok, was not sure, thanks 👍